configure

```diff
@@ -144,13 +144,13 @@ fi
 # and then move it.
 # Note that the resulting chplconfig is expected to cover the same
 # material as any existing chplconfig.
-./util/printchplenv --overrides --anonymize > configured-chplconfig
+./util/printchplenv --simple --no-tidy --overrides --anonymize > configured-chplconfig
```

Comment:
It seems to me that we want `--tidy` for configure - if a variable isn't relevant, wouldn't we prefer to leave the default value? (Additionally it seems hard to believe there would be an overridden variable that is not relevant.. but maybe I'm just having a failure of imagination.).

Reply:
I wasn't sure here, but assumed `--no-tidy` would be safer somehow. I am also failing to imagine a scenario where it would matter. I'll remove the `--no-tidy`.